### PR TITLE
Reload/ReloadLauncher - Add notifications when buddy loading or linking

### PR DESCRIPTION
**Pull request by Drofseh**
**When merged this pull request will:**
- Add notifications to reloadLauncher
- Also set _reloadTime to `magazineReloadTime min 2.5` instead of just 2.5 if there is no `buddyReloadTime` set

- [x] If the contribution affects [the documentation](https://github.com/acemod/ACE3/tree/master/docs), please include your changes in this pull request so the documentation will appear on the [website](https://ace3.acemod.org/).
- [x] [Development Guidelines](https://ace3.acemod.org/wiki/development/) are read, understood and applied.
- [x] Title of this PR uses our standard template `Component - Add|Fix|Improve|Change|Make|Remove {changes}`.

### CODE REVIEW

File: addons/reloadlaunchers/stringtable.xml [hunk 1]
```diff
@@ -18,6 +18,12 @@
             <Chinese>裝載發射器</Chinese>
             <Turkish>Fırlatıcıyı Yükle</Turkish>
         </Key>
+        <Key ID="STR_ACE_ReloadLaunchers_LoadingStarted">
+            <English>is loading your launcher.</English>
+        </Key>
+        <Key ID="STR_ACE_ReloadLaunchers_LoadingAborted">
+            <English>stopped loading your launcher.</English>
+        </Key>
```
Comment (BrettMayson) [inline addons/reloadlaunchers/stringtable.xml:26]: This is not going to work for some languages, even though in this case I think it should for every language Arma supports. Not sure if we do this anywhere else, but probably want something like `%1 is loading your launcher`, and format on that.

**Review by BrettMayson — COMMENTED** [review]
Personally I'd like an option to disable this, but I think it could be enabled by default

File: addons/reloadlaunchers/stringtable.xml (hunk 1 above)
Comment (Drofseh) [reply addons/reloadlaunchers/stringtable.xml:26]: Yeah I can format it like that instead.

**Comment by Drofseh** [comment]
> Personally I'd like an option to disable this, but I think it could be enabled by default

There are already messages for the loader/linker that give the status, but I don't mind adding a setting if you're really like.

**Comment by BrettMayson** [comment]
In my opinion no reason not to add an option, but I'm fine if we don't want to. I just personally prefer having people say "I'm loading your launcher". Notifications about your own actions don't really matter as much.

**Comment by Drofseh** [comment]
Merged into #9335 instead

**Pull request closed without merging**